io.github.spacingbat3.webcord.yml

```diff
@@ -0,0 +1,50 @@
+app-id: io.github.spacingbat3.webcord
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: run.sh
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --device=all
+modules:
+  - name: webcord
+    buildsystem: simple
+    sources:
+      - type: archive
+        url: https://github.com/vega-d/WebCord_flatpak_build/releases/download/v0.0-rolling/WebCord_tarball.tar.gz
```

Comment:
I generally agree that this should be versioned somehow (even by using the build date). We want manifests to be reproducible.

Reply:
This was already discussed in https://github.com/flathub/flathub/pull/3229#pullrequestreview-993984438
Once the app gets merged in flathub we will change it to versioned releases